docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -460,47 +528,47 @@ Then reboot the flight controller by selecting "Reboot Vehicle" from the "Tools"
 
 Re-Configuring PX4 DSHOT Outputs
 ################################
-.. note:: Only needed on some flight controlllers, see the `ArduCopter <https://ardupilot.org/copter/docs/common-brushless-escs.html#mixing-esc-protocols>`_ and `PX4 <https://docs.px4.io/master/en/peripherals/dshot.html#wiring-connections>`_ documentation for more details on which
+.. note:: These steps are only needed on some flight controlllers, see the `ArduCopter <https://ardupilot.org/copter/docs/common-brushless-escs.html#mixing-esc-protocols>`_ and `PX4 <https://docs.px4.io/master/en/peripherals/dshot.html#wiring-connections>`_ documentation for more details on which
```

Comment:
These steps are necessary on only some flight controllers. See the <links> documentation for more details on if your flight controller is included.